ev3dev/dc_motor.go

```diff
@@ -89,43 +76,18 @@ func (m *DCMotor) Command(comm string) *DCMotor {
 		m.err = fmt.Errorf("ev3dev: command %q not available for %s (available:%q)", comm, m, avail)
 		return m
 	}
-	err = m.writeFile(fmt.Sprintf(DCMotorPath+"/%s/"+command, m), comm)
-	if err != nil {
-		m.err = fmt.Errorf("ev3dev: failed to issue dc-motor command: %v", err)
-	}
+	m.err = setAttribute(m, command, comm)
 	return m
 }
 
 // DutyCycle returns the current duty cycle value for the DCMotor.
 func (m *DCMotor) DutyCycle() (int, error) {
-	if m.err != nil {
-		return -1, m.Err()
-	}
-	b, err := ioutil.ReadFile(fmt.Sprintf(DCMotorPath+"/%s/"+dutyCycle, m))
-	if err != nil {
-		return -1, fmt.Errorf("ev3dev: failed to read duty cycle: %v", err)
-	}
-	sp, err := strconv.Atoi(string(chomp(b)))
-	if err != nil {
-		return -1, fmt.Errorf("ev3dev: failed to parse duty cycle: %v", err)
-	}
-	return sp, nil
+	return intAttr(attribute(m, dutyCycle))
 }
 
 // DutyCycleSetpoint returns the current duty cycle set point value for the DCMotor.
```

Comment:
Since the API treats (correctly) setpoint as one word, the documentation should follow suit.